README.md

```diff
@@ -253,46 +249,49 @@ Like `attr`, the `defaultValue` option can be a function that is invoked to gene
 
 ```javascript
 // app/models/person.js
-import Model from 'ember-data/model';
+
+import Model, { attr, belongsTo, hasMany } from '@ember-data/model';
 import { fragment } from 'ember-data-model-fragments/attributes';
 
-export default Model.extend({
-  name: fragment('name', {
+export default class PersonModel extends Model {
+  @fragment('name', {
     defaultValue() {
       return {
         first: 'Unsullied',
-        last: Ember.uuid()
-      }
+        last: Ember.uuid(),
```

Comment:
Should probably not use a global Ember in an example. Can import `uuid` and use that (make a comment about installing uuid from npm)

Reply:
Added the `uuid` note, though it still seems kind of awkward. Do you know if it's possible to use `guidFor` here?

`import { guidFor } from '@ember/object/internals';`

Reply:
Maybe just change the example 🙂